tests/transports-integrations/tests/integrations/test_google.py

```
@@ -0,0 +1,466 @@
+    skip_if_no_api_key,
+    COMPARISON_KEYWORDS,
+    WEATHER_KEYWORDS,
+    LOCATION_KEYWORDS,
+)
+from ..utils.config_loader import get_model
+
+
+@pytest.fixture
+def google_client():
+    """Configure Google GenAI client for testing"""
+    from ..utils.config_loader import get_integration_url, get_model
```

Pylint (3.3.7)

[warning] Redefining name 'get_model' from outer scope (line 43)

(W0621)